src/features/common/Layout/Navbar/index.tsx

```diff
@@ -216,7 +216,10 @@ export default function NavbarComponent(props: any) {
                         <a
                           key={submenu.title}
                           className={'menuRow'}
-                          href={submenu.onclick}
+                          href={i18n.language === 'en' ? submenu.onclickEN :
+                          i18n.language === 'de' ? submenu.onclickDE :
+                          i18n.language === 'es' ? submenu.onclickES :
+                        ''}
```

Comment:
This leads for wrong links (just https://planet-webapp-8c27paq4u-planetapp.vercel.app/) for French, Italian and Portuguese. There should be a default link without a locale if no i18n.language matches (as maybe we add further languages in the future).